tests/unit/test_charm.py

```diff
@@ -915,3 +915,214 @@ def test_postgres_patch_storage_config_defaults_to_database_relation(self):
         }
         environment = plan.to_dict()["services"]["livepatch"]["environment"]
         self.assertEqual(environment, environment | required_environment)
+
+    def test_pro_airgapped_server_relation__success(self):
+        """Test pro-airgapped-server relation."""
+        self.harness.set_leader(True)
+        self.harness.enable_hooks()
+
+        self.harness.charm._state.dsn = "postgresql://123"
+        self.harness.charm._state.resource_token = TEST_TOKEN
+
+        with patch("src.charm.LivepatchCharm.migration_is_required") as migration:
+            migration.return_value = False
+            self.harness.update_config(
+                {
+                    "server.url-template": "http://localhost/{filename}",
+                    "server.is-hosted": False,
+                }
+            )
```

Comment:
Why do we need this?

Reply:
This is what we do in other test cases. The charm config has to be updated to make sure the required ones are set and we get to see the Pebble plan changes.